### DISCUSSION TITLE
CSV import / export [May 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5651] [Discord Threads]
> 🙋‍♂️ **v.plyndin**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### <@1067017773538099200> You uploaded data to NocoDB using a CSV file. Made some changes to data. Downloaded CSV. Please confirm if this is scenario you tried.   
####    
#### Likely scenario, the column was hidden. You can open `fields` in NocoDB menu & ensure all the fields you need are enabled. If its a system field (Id in your case), you can also enable `Show system fields` option. Do let us know if our query understanding was right & if it helps  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1093877170910269500)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
